packages/python/plotly/_plotly_utils/utils.py

```diff
@@ -40,10 +40,15 @@ def encode(self, o):
         Note that setting invalid separators will cause a failure at this step.
 
         """
-
         # this will raise errors in a normal-expected way
+        self.hasinfnans = False
```

Comment:
Are you using `hasinfnans` anywhere else now?

Reply:
no you were right :-). Thanks!